sct-commons/src/main/java/org/lfenergy/compas/sct/commons/LnService.java

```diff
@@ -146,13 +146,14 @@ public void updateOrCreateDOAndDAInstances(TAnyLN tAnyLN, DoLinkedToDa doLinkedT
                 });
     }
 
-    public void completeFromDAInstance(TIED tied, String ldInst, TAnyLN anyLN, DoLinkedToDa doLinkedToDa) {
+    public DoLinkedToDa completeFromDAInstance(TIED tied, String ldInst, TAnyLN anyLN, DoLinkedToDa doLinkedToDa) {
```

Comment:
I suggest change method name  Example: `getCompletedDoLinkedToDaFromDAInstance`

Reply:
I think it can be part of the te interface that `LnService` implement

Reply:
- OK for `getDoLinkedToDaCompletedFromDAI`
- OK to add method to `LnEditor`